sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -865,6 +865,7 @@ def test_corrwith_bad_axis(self):
     self._run_error_test(lambda df: df.corrwith(df, axis=5), df)
 
   @unittest.skipIf(PD_VERSION < (1, 2), "na_action added in pandas 1.2.0")
+  @unittest.skipIf(PD_VERSION >= (2, 1), "applymap deprecated in 2.1.0")
```

Comment:
is there harm to keep testing applymap until we remove the functionality?

Reply:
I'm just trying to clean up deprecation warnings. I could suppress that warning instead.

Reply:
i would keep the test in this case if we care about it, unless we remove support from Beam SDK for it.

Reply:
Suppressed the warning instead.